.travis.yml

```diff
@@ -0,0 +1,19 @@
+group: edge
+
+services:
+  - docker
+
+on:
+  branch: master
+
+env:
+  global:
+  - DOCKER_COMPOSE_VERSION=1.21.0
```

Comment:
Do you have any evidence this environement variable is used by travis? As far as I can see, it is not and should not be needed anyway.